Sources/ResilientDecoding/ResilientPropertyLevelErrors.swift

```diff
@@ -0,0 +1,156 @@
+// Created by George Leontiev on 4/3/20.
+// Copyright © 2020 Airbnb Inc. All rights reserved.
+
+import Foundation
+
+/**
+ The `Resilient` property wrapper has a `projectedValue` in `DEBUG` builds which contains the error(s) encountered when decoding that property. This is intended to only be used during development, error reporting in release should instead be done via `ResilientDecodingErrorReporter`.
+ Limiting this functionality to `DEBUG` allows us to trade some extra complexity in this file for better _clarity at point of use_ (https://swift.org/documentation/api-design-guidelines/) and guarantee that this doesn't impact release builds.
+ */
```

Comment:
Should we put this comment on `PropertyLevelErrors` instead so it shows up in Xcode?